solidity/contracts/NetworkSettings.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+import "./INetworkSettings.sol";
+import "./utility/Owned.sol";
+import "./utility/Utils.sol";
+
+/**
+ * @dev This contract maintains the network settings.
+ */
+contract NetworkSettings is INetworkSettings, Owned, Utils {
+    address private _networkFeeWallet;
```

Comment:
still need specific getters for these, along with a description for what which one is (network fee = portion that's taken from all swap fees etc.)

Reply:
We have a getter, it's called `feeParams`.

Reply:
I know, I want dedicated getters as well

Reply:
Done.